docs/plugins/image.md

```diff
@@ -1,77 +1,81 @@
 # Image
 
-The standard way of displaying images in Lightning is to just specify the `src`. This is the prefered way for your App's local assets (such as background, splash screen, logo and icons).
+The standard way of displaying images in Lightning is to just specify the `src`. This is the preferred way for your local assets (such as background, splash screen, logo and icons) of Lightning Apps.
```

Comment:
no need for "your" here I think (your local assets => local assets)

Reply:
Agreed.